cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1257,6 +1262,66 @@ def make_resource(self, data, **kwargs):
         return HeadNode(**data)
 
 
+class LoginNodeImageSchema(BaseSchema):
+    custom_ami = fields.Str(validate=validate.Regexp(r"^ami-[0-9a-z]{8}$|^ami-[0-9a-z]{17}$"))
+
+    @post_load
+    def make_resource(self, data, **kwargs):
+        """Generate resource."""
+        return LoginNodeImage(**data)
+
+
+class LoginNodeSshSchema(BaseSchema):
+    key_name = fields.Str(required=True)
+
+    @post_load
+    def make_resource(self, data, **kwargs):
+        """Generate resource."""
+        return LoginNodeSsh(**data)
+
+
+class LoginNodeNetworkingSchema(BaseNetworkingSchema):
+    subnet_id = fields.Str(required=True)
+    elastic_ip = fields.Str()
+    proxy = fields.Str()
+
+    @post_load
+    def make_resource(self, data, **kwargs):
+        """Generate resource."""
+        return LoginNodeNetworking(**data)
+
+
+class LoginNodePoolSchema(BaseSchema):
+    name = fields.Str(required=True, validate=[validate.Length(min=1), validate.Regexp(r'^[a-z0-9]+$')])
+    instance_type = fields.Str(required=True)
+    image = fields.Nested(LoginNodeImageSchema)
+    networking = fields.Nested(LoginNodeNetworkingSchema, required=True)
+    count = fields.Int(required=True, validate=validate.Range(min=1))
+    ssh = fields.Nested(LoginNodeSshSchema, required=True)
+    iam = fields.Nested(IamSchema)
+    gracetime_period = fields.Int(validate=validate.Range(max=120))
+    admin_user = fields.Str(required=True)
```

Comment:
Let's remove these parameters for now (let's sync on this).

Reply:
Done! I have removed them.